developers_chamber/qa/checks.py

```diff
@@ -79,3 +79,21 @@ def _run_check(self):
         wrong_import_order_files = set(changed_files) & set([diff.b_path for diff in self._get_unstaged()])
         if wrong_import_order_files:
             raise QAError('Found unsorted import(s) in following file(s):', '\n'.join(wrong_import_order_files))
+
+
+class TestMethodNamesQACheck(QACheck):
+    """
+    Checks that test methods are named correctly (i.e. does not begin with "test_should").
+    """
+    name = 'Check test method names'
+
+    def _run_check(self):
+        disallowed_method_names = []
+        for diff_obj in self._get_diffs():
+            for line in diff_obj.diff.decode().split('\n'):
+                match = re.search(r'^\+ +def (test_should_[^\(]+)\(', line)
```

Comment:
you are looking in all changes not only in tests modules?

Better solution will be try to find valid test names "test_[^\(]+)_should_[^\(]+)

Reply:
Yes, searching test-only modules would be way more difficult. The test method signature is quite unique I think, so there's no need for that.

Why do you think it's better to find valid test names? I want to find invalid ones and list them to the user.

Reply:
ok